### PR TITLE
Cubed-xarray post

### PR DESCRIPTION
**Pull request by TomNicholas**

### CODE REVIEW

**Comment by TomNicholas** [comment]
@andersy005 @jhamman can one of you authorize vercel on this please so I can see if it renders everything properly?

I guess I probably have to put the images in separately actually don't I...

File: src/posts/cubed-xarray/index.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+You can see again that the [projected memory](https://tom-e-white.com/cubed/user-guide/memory.html) usage is below the allowed memory usage (else Cubed would have raised an exception before the job even started running), and the actual peak memory used was lower still. We've also plotted the reserved memory, which is a parameter intended to account for the memory usage of the executor itself (i.e. Lithops here), and was estimated by measuring beforehand using [`cubed.measure_reserved_memory()`](https://tom-e-white.com/cubed/generated/cubed.measure_reserved_memory.html#cubed.measure_reserved_memory).
+
+One obvious tradeoff for this memory stability is that Cubed took **a lot longer** to complete - roughly 4x longer then dask for the 1.5TB workload (45m 22s vs 11m 26s). We will come back to discuss this shortly.
+
+Finally it's interesting to look at Cubed's equivalent of the task graph. To calculate one array (the product $<UV>_t$ from the quadratic means problem), Cubed's "Plan" for processing 1.5TB of data looks like this:
```
Comment (andersy005) [inline src/posts/cubed-xarray/index.md:155]: `$<UV>_t$` is problematic... it results in this error

```
SyntaxError: unknown: Expected corresponding JSX closing tag for <UV>.
```

Comment (TomNicholas) [reply src/posts/cubed-xarray/index.md]: Okay, is latex formatting allowed? I just tried changing it to `$UV$` to see if that helps

Comment (andersy005) [reply src/posts/cubed-xarray/index.md]: a subset of it is supported. i intend to add the necessary plugins required to support the missing features

**Comment by andersy005** [comment]
@TomNicholas, it's working: https://xarray-3eyffzaae-xarray.vercel.app/blog/cubed-xarray :)

**Comment by TomNicholas** [comment]
Thanks @andersy005! I just need to double-check whether this requires revisions, then can I merge it myself and have it deploy?

File: src/posts/cubed-xarray/index.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
+---
+title: 'Unit-aware arithmetic in Xarray, via pint'
```
Comment (andersy005) [inline src/posts/cubed-xarray/index.md:2]: this needs to be changed, right?

Comment (TomNicholas) [reply src/posts/cubed-xarray/index.md]: Yes! Just changed.

**Comment by andersy005** [comment]
> I just need to double-check whether this requires any last-minute revisions, then can I merge it myself and have it deploy?

yep. once the PR is merged, vercel will automatically redeploy the main site

**Pull request merged**